doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_application.rst

```diff
@@ -0,0 +1,138 @@
+The following is a summary of the most important ones:
+
+Updated |NCS| toolchain
+  The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+  See :ref:`migration_cs3_to_2_6_99_cs2_env` for more info on how to upgrade your current |NCS| installation based on version 2.4.99-cs3.
+
+SDFW and SCFW
+  The Secure Domain Firmware (SDFW) and System Controller Firmware (SCFW) are no longer built from the source during the application build process, but they are installed as binaries from the provided firmware bundle and programmed when flashing the application to the device.
+  For additional details, see :ref:`migration_cs3_to_2_6_99_cs2_env_bringup`.
+
+nRF Util has now replaced nRF Command Line Tools.
+  nRF Util is now the main command line backend utility.
+
+Updated boards
+  SOC1-based boards have been removed and FP1-based board have been added.
+  The board names for the Application, Radio, and PPR core have been updated.
+  For Additional details, see `migration_cs3_to_2_6_99_cs2_app_general`_
+
+DTS changes
+  The layout of DTS files and the names of DTS nodes related to the updated board names have been also updated, also affecting overlay files from applications and samples.
+  If your application required a specific custom board, you must update the custom board files to match the changes done to the nRF54H20 SoC DTS files.
+
+Lifecycle State changes
+  To correctly operate the nRF54H20 DK, it must be set in LCS ``RoT``.
+  For additional details, see :ref:`migration_cs3_to_2_6_99_cs2_env_lcsrot`.
+  Also, it is no longer possible to perform an unauthenticated backward LCS transitions.
+  This means that once set to ``RoT``, it is no longer possible to revert to LCS state ``EMPTY``.
+
+.. note::
+   Not all the features supported in |NCS| v2.4.99-cs3 have been ported to v2.6.99-cs2 (for example, the nRF54H20 support for the nRF Desktop and nRF Machine Learning applications).
+   Additional features will be provided in future releases.
+
+Required changes
+****************
+
+The following changes are mandatory to make your application work in the same way as in previous releases.
+
+This section describes the changes related to samples and applications.
+
+Samples and applications
+========================
+
+.. _migration_cs3_to_2_6_99_cs2_app_general:
+
+General
+-------
+
+.. toggle::
+
+   * The nRF54H20 DK samples and applications are now using the following FP1-based board names:
+
+     * Application core: ``nrf54h20dk_nrf54h20_cpuapp`` (Previously ``nrf54h20pdk_nrf54h20_cpuapp@soc1``)
+     * Radio core: ``nrf54h20dk_nrf54h20_cpurad`` (Previously ``nrf54h20pdk_nrf54h20_cpurad@soc1``)
+     * PPR core: ``nrf54h20dk_nrf54h20_cpuppr`` (Previously ``nrf54h20pdk_nrf54h20_cpuppr@soc1``)
+
+     The previously used SOC1-based board files have been removed.
+
+Security
+========
```

Comment:
```suggestion
--------
```
I think this should be on the same level as the General heading. Not sure if the same is true of the zcbor heading.